app/screens/home/channel_list/categories_list/index.tsx

```diff
@@ -61,18 +59,14 @@ const CategoriesList = ({channelsCount, currentTeamId, iconPad, isCRTEnabled, is
 
     let content;
 
-    if (!currentTeamId) {
-        content = (<LoadTeamsError/>);
-    } else if (channelsCount < 1) {
-        content = (<LoadChannelsError teamId={currentTeamId}/>);
+    if (channelsCount < 1) {
+        content = (<LoadChannelsError/>);
```

Comment:
🤔  we could have multiple channels in the database but that does not mean that a team has been selected.

Also, it looks like `channelsCount` is not really about the current team but all channels in total regardless of the team